docs/_basic/listening_modals.md

```diff
@@ -69,3 +71,23 @@ app.view('view_b', async ({ ack, body, view, client }) => {
 
 });
 ```
+
+---
+
+##### handle view closed events
+
+The built-in `view()` method can also handle `view_closed` events, which are sent when a <a href="https://api.slack.com/reference/block-kit/views">view payload</a> has the `notify_on_close` flag set.
+The event contains the respective `callback_id` of the closed view.
```

Comment:
I think we may not need this line!

Reply:
I agree as this is now mentioned also in the overall view section - will remove it.